src/core/public/chrome/ui/header/header_help_menu.tsx

```diff
@@ -122,6 +122,7 @@ interface Props {
   opensearchDashboardsVersion: string;
   useDefaultContent?: boolean;
   opensearchDashboardsDocLink: string;
+  survey?: string;
```

Comment:
nit:

line `124` says `opensearchDashboardsDocLink` then for consistency I think it should be `surveyLink` if this value is the link.